### ISSUE TITLE
Download from Premiumize is not happening

### ISSUE THREAD

**Issue by arthursoares** [piejanssens/premiumizer#208]
Hello! 

I'm running on a RaspberryPi, the script connect and is able to monitor the state of Downloads in Premiumize, but it fails to download the file from there.

![Screen Shot 2019-07-26 at 14 37 38](https://user-images.githubusercontent.com/533339/61952012-03d7ec80-afb3-11e9-92eb-26c2e2aa1d02.png)
![Screen Shot 2019-07-26 at 14 37 25](https://user-images.githubusercontent.com/533339/61952013-03d7ec80-afb3-11e9-80ac-47a2cb49aed5.png)
![Screen Shot 2019-07-26 at 14 37 20](https://user-images.githubusercontent.com/533339/61952014-03d7ec80-afb3-11e9-85bf-4c51b18c9b7a.png)

Here are some screenshots. Nothing was pointed out in the logs.

Anything else I can help to try to debug this? Cheers and thank you for the great app :)

**Comment by neox387** [comment]
And what does the home screen look like, I think the task do not have a category ? so it does not start the download..

did you manually add them or trough watchfolder ?

btw that scan watchdir folder regularly doesn't need to be enabled only if it doesn not work or for network share

**Comment by arthursoares** [comment]
Adding the Categories worked out quite well :) I think that was the issue. Thank you.

tl;dr: I added a 'default' category and enabled the "Download everything and assign the category 'default'" and it works.

**Comment by neox387** [comment]
Mm toy already had the option Download everything enabled I see so should have worked without making the default category didn't see before.

Will test it later to see if it's broken =D

**Comment by arthursoares** [comment]
@neox387 

Yeah, it didn't work without adding a category 'default' to the list.

Also, some behavior changed with the new update. Same settings, but on my home screen it says: "Downloads are disabled"


```
07-29 18:22:55       apscheduler.scheduler                    : DEBUG    : Looking for jobs to run
07-29 18:22:55       apscheduler.scheduler                    : DEBUG    : Next wakeup is due at 2019-07-29 18:22:56.232089+02:00 (in 0.993541 seconds)
07-29 18:22:55       geventwebsocket.handler                  : DEBUG    : Initializing WebSocket
07-29 18:22:55       geventwebsocket.handler                  : DEBUG    : Validating WebSocket request
07-29 18:22:55       geventwebsocket.handler                  : DEBUG    : Initializing WebSocket
07-29 18:22:55       geventwebsocket.handler                  : DEBUG    : Validating WebSocket request
07-29 18:22:55       geventwebsocket.handler                  : DEBUG    : Attempting to upgrade connection
07-29 18:22:55       geventwebsocket.handler                  : DEBUG    : WebSocket request accepted, switching protocols
07-29 18:22:55       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Received request to upgrade to websocket
07-29 18:22:55       geventwebsocket.handler                  : INFO     : 192.168.178.20 - - [2019-07-29 18:22:55] "GET /socket.io/?EIO=3&transport=polling&t=Mm-Yhew&sid=fef9f808e1e14634a7b2ef79dccf21bc HTTP/1.1" 200 191 0.007877
07-29 18:22:55       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Upgrade to websocket successful
07-29 18:22:56       apscheduler.scheduler                    : DEBUG    : Looking for jobs to run
07-29 18:22:56       apscheduler.scheduler                    : DEBUG    : Next wakeup is due at 2019-07-29 18:22:57.232089+02:00 (in 0.991489 seconds)
07-29 18:22:56       apscheduler.executors.default            : INFO     : Running job "update (trigger: interval[0:00:01], next run at: 2019-07-29 18:22:57 CEST)" (scheduled at 2019-07-29 18:22:56.232089+02:00)
07-29 18:22:56       root                                     : DEBUG    : def update started
07-29 18:22:56       root                                     : DEBUG    : def prem_connection started
07-29 18:22:56       urllib3.connectionpool                   : DEBUG    : https://www.premiumize.me:443 "POST /api/transfer/list HTTP/1.1" 200 None
07-29 18:22:56       root                                     : DEBUG    : def parse_task started
07-29 18:22:56       root                                     : DEBUG    : def get_task started
07-29 18:22:56       socketio.server                          : INFO     : emitting event "update_task" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"l8GErK4nFJ45Lq88W3HOxQ","folder_id":"WUWFAcCTW0JJ65XoIUxgNA","file_id":null,"name":"12 Angry Men (1957) [BluRay] [1080p] [YTS.AM]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"l8GErK4nFJ45Lq88W3HOxQ","folder_id":"WUWFAcCTW0JJ65XoIUxgNA","file_id":null,"name":"12 Angry Men (1957) [BluRay] [1080p] [YTS.AM]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       root                                     : DEBUG    : def get_task started
07-29 18:22:56       socketio.server                          : INFO     : emitting event "update_task" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"H4RWAmnkG-R0PrxyQf7o1g","folder_id":"96hDAC2SBe-MDKrffb91aw","file_id":"lCX9vPXWf_fgRwMcGfMf-g","name":"Roma.2018.2160p.NF.WEBRip.x265.10bit.HDR.Master5.mkv","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"H4RWAmnkG-R0PrxyQf7o1g","folder_id":"96hDAC2SBe-MDKrffb91aw","file_id":"lCX9vPXWf_fgRwMcGfMf-g","name":"Roma.2018.2160p.NF.WEBRip.x265.10bit.HDR.Master5.mkv","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       root                                     : DEBUG    : def get_task started
07-29 18:22:56       socketio.server                          : INFO     : emitting event "update_task" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"ffiYID8OdyLtaJqIK5FXnw","folder_id":"BZQgjdhU11T8Cb9W0MVvZA","file_id":null,"name":"At.Eternitys.Gate.2018.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-FGT","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"ffiYID8OdyLtaJqIK5FXnw","folder_id":"BZQgjdhU11T8Cb9W0MVvZA","file_id":null,"name":"At.Eternitys.Gate.2018.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-FGT","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       root                                     : DEBUG    : def get_task started
07-29 18:22:56       socketio.server                          : INFO     : emitting event "update_task" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"2BZ_6k8GiCfh4Jc0SHODKA","folder_id":"BDztI7hBUwYQ0RTRb1U9wQ","file_id":null,"name":"Minding.The.Gap.2018.1080p.WEB.H264-AMRAP[rarbg]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"2BZ_6k8GiCfh4Jc0SHODKA","folder_id":"BDztI7hBUwYQ0RTRb1U9wQ","file_id":null,"name":"Minding.The.Gap.2018.1080p.WEB.H264-AMRAP[rarbg]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       root                                     : DEBUG    : def get_task started
07-29 18:22:56       socketio.server                          : INFO     : emitting event "update_task" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"-51OaZiHJTWEk_9nxJjOMg","folder_id":"b5sUpGip2mYUbWJex7S9sA","file_id":null,"name":"Beautiful Boy (2018) [BluRay] [1080p] [YTS.AM]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"movies"}}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"-51OaZiHJTWEk_9nxJjOMg","folder_id":"b5sUpGip2mYUbWJex7S9sA","file_id":null,"name":"Beautiful Boy (2018) [BluRay] [1080p] [YTS.AM]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"movies"}}]
07-29 18:22:56       root                                     : DEBUG    : def get_task started
07-29 18:22:56       socketio.server                          : INFO     : emitting event "update_task" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"ekWjL545a_Zdga1PaDuNVw","folder_id":"bimbdrdop_HGUtTBlPpFEA","file_id":null,"name":"Prospect.2018.2160p.BluRay.REMUX.HEVC.DTS-HD.MA.5.1-FGT","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"ekWjL545a_Zdga1PaDuNVw","folder_id":"bimbdrdop_HGUtTBlPpFEA","file_id":null,"name":"Prospect.2018.2160p.BluRay.REMUX.HEVC.DTS-HD.MA.5.1-FGT","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       root                                     : DEBUG    : def get_task started
07-29 18:22:56       socketio.server                          : INFO     : emitting event "update_task" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"Hca42tEp7FGzQW-19z2CIw","folder_id":"UPFPIXRWPbOoEDYayscITA","file_id":null,"name":"Avengers Endgame.2019.V2.1080p.HDRip.X264.AC3-EVO[EtHD]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"Hca42tEp7FGzQW-19z2CIw","folder_id":"UPFPIXRWPbOoEDYayscITA","file_id":null,"name":"Avengers Endgame.2019.V2.1080p.HDRip.X264.AC3-EVO[EtHD]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]
07-29 18:22:56       socketio.server                          : INFO     : emitting event "tasks_updated" to all [/]
07-29 18:22:56       engineio.server                          : INFO     : 5db9e2c8990a462e9c54c56e430d800a: Sending packet MESSAGE data 2["tasks_updated",{}]
07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["tasks_updated",{}]
07-29 18:22:56       apscheduler.executors.default            : INFO     : Job "update (trigger: interval[0:00:10], next run at: 2019-07-29 18:23:06 CEST)" executed successfully
07-29 18:22:56       apscheduler.scheduler                    : DEBUG    : Looking for jobs to run
07-29 18:22:56       apscheduler.scheduler                    : DEBUG    : Next wakeup is due at 2019-07-29 18:23:06.347271+02:00 (in 9.995746 seconds)
07-29 18:22:57       geventwebsocket.handler                  : DEBUG    : Initializing WebSocket
07-29 18:22:57       geventwebsocket.handler                  : DEBUG    : Validating WebSocket request
```

Somehow it is passing this flag:

`07-29 18:22:56       engineio.server                          : INFO     : fef9f808e1e14634a7b2ef79dccf21bc: Sending packet MESSAGE data 2["update_task",{"task":{"progress":100,"speed":null,"dlsize":"","eta":" ","id":"Hca42tEp7FGzQW-19z2CIw","folder_id":"UPFPIXRWPbOoEDYayscITA","file_id":null,"name":"Avengers Endgame.2019.V2.1080p.HDRip.X264.AC3-EVO[EtHD]","type":"Torrent","cloud_status":"finished","local_status":"download_disabled","category":"default"}}]`

`"local_status":"download_disabled"`

**Comment by neox387** [comment]
So it still does not download then?

**Comment by arthursoares** [comment]
@neox387 
it downloaded one movie, but then all others had that status:

```
[downloads]
download_enabled = 1
download_all = 1
download_rss = 0
download_max = 2
download_threads = 4
download_speed = -1
remove_cloud = 1
remove_cloud_delay = 1
seed_torrent = 0
download_location = /media/pi/klarna_i
nzbtomedia_location = [Change to path]\nzbToMedia.py
jd_enabled = 0
jd_username =
jd_password =
jd_device_name =
aria2_enabled = 0
aria2_host = localhost
aria2_port = 6800
aria2_secret = premiumizer
```

this is my settings.cfg

**Comment by arthursoares** [comment]
update: 
when I added a new torrent on Premiumize, it works. But all other Transfers still have that status.

**Comment by neox387** [comment]
everything should be fixed with next PR #213

I now made it default so category 6 has the name default.
